ledger-api/perf-testing/src/main/scala/com/digitalasset/ledger/api/perf/util/PerformanceTest.scala

```diff
@@ -49,8 +49,8 @@ abstract class PerformanceTest extends Bench[Double] {
 
   @transient lazy val persistor = Persistor.None
 
-  protected def daConfig: Array[org.scalameter.KeyValue] =
-    Array(
+  protected def daConfig: Seq[org.scalameter.KeyValue] =
+    Seq[org.scalameter.KeyValue](
```

Comment:
Probably not particularly important, but should we use some sort of `IndexedSeq` here instead, since we are replacing an `Array`?

Reply:
The only use is in https://github.com/digital-asset/daml/blob/57217f33e8f5d0fa1944c4c6698c42bd2aab66b3/ledger-api/rs-grpc-akka/src/perf/scala/com/digitalasset/grpc/adapter/ServerStreamingBenchmark.scala#L34 which seemed to fall under the category of “I really don’t care what data structure we use here” which means I generally default to `Seq` but happy to use something else.

Reply:
The `Seq` should actually be better in this case, [IIRC](https://stackoverflow.com/a/54323858/3314107). Thanks for pointing that out.